packages/fx-core/src/common/deps-checker/internal/funcToolChecker.ts

```diff
@@ -82,6 +82,9 @@ export class FuncToolChecker implements DepsChecker {
       this._telemetry.sendEvent(DepsCheckerEvent.funcAlreadyInstalled, {
         "global-func-version": `${await this.queryGlobalFuncVersion()}`,
       });
+      if (!isPortableFuncInstalled) {
+        await this.cleanup();
```

Comment:
Will `cleanup` throw error if portable folder not exist?

Reply:
Not.
![image](https://user-images.githubusercontent.com/12523371/169637806-9f919396-8a86-4cb6-b205-05ab4c5d857a.png)